src/main/java/org/hiero/sketch/remoting/SketchClientActor.java

```diff
@@ -0,0 +1,130 @@
+package org.hiero.sketch.remoting;
+
+import akka.actor.AbstractActor;
+import akka.actor.ActorRef;
+import akka.actor.ActorSelection;
+import akka.actor.Props;
+import akka.japi.pf.ReceiveBuilder;
+import org.hiero.sketch.dataset.RemoteDataSet;
+import org.hiero.sketch.dataset.api.IDataSet;
+import org.hiero.sketch.dataset.api.PartialResult;
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * This actor wires a local observable pertaining to a map or sketch
+ * operation with its remote execution. Submitting a {Map,Sketch}Operation
+ * to this actor returns an observable that streams values in accordance
+ * to its execution on the remote node.
+ */
+public class SketchClientActor extends AbstractActor {
```

Comment:
We'll also have to figure out how to GC these guys. Do you know how the system keeps track of actors?
We'd like for actors without references to be GC-ed. At the same time they should also initiate the GC of the remote counterparts.